### DISCUSSION TITLE
tPRC request from rsc by undefined,

### DISCUSSION THREAD

**Discussion by Zeff01** [t3-oss/create-t3-turbo#1134] [General]
![image](https://github.com/user-attachments/assets/a276d6ef-220c-41c6-a71e-29b4dbb82353)

**Comment by Zeff01** [comment]
Anyone have experience this error in setting the repo?

**Reply by daniel0barbosa** [reply]
i have the same problem, my db is working fine, and deploy vercel too, but i don't resolve this.Just comment to up

**Reply by daniel0barbosa** [reply]
@Zeff01 you resolved this?
